src/svyboxplot.jl

```diff
@@ -0,0 +1,29 @@
+"""
+```
+	svyboxplot(design, x, y; kwargs...)
+```
+Box plot of survey design variable `y` grouped by column `x`.
+
+Weights can be specified by a Symbol using the keyword argument `weights`.
+
+The keyword arguments are all the arguments that can be passed to `mapping` in
+[AlgebraOfGraphics](https://docs.juliahub.com/AlgebraOfGraphics/CHIaw/0.4.7/).
+
+```@example svyboxplot
+julia> using survey
+
+julia> data(api);
+
+julia> dstrat = svydesign(data = apistrat, id = :1, strata = :stype, weights = :pw, fpc = :fpc);
+
+julia> bp = svyboxplot(dstrat, :stype, :enroll; weights = :pw)
```

Comment:
I think `svyboxplot` can pick the weights from the design. No need to pass the weights if you are passing the design.

Reply:
You mean get the weights from the "probs" column of `design.variables`? That's a good idea.

Reply:
Since many functions require frequency weights, we can store it in `design.variables` rather than having each function derive it from `probs`. 

Instead of `probs`, perhaps, we should call it `probability_weights_reserved`? Similarly, `frequency_weights_reserved`.

The reason we're only storing probability weights is that we didn't anticipate so many Julia functions to use frequency weights. I believe the `survey` package in R only store probability weights, and most R functions use probability weights.